app/src/main/java/com/projectfkklp/saristorepos/activities/dashboard/DashboardTodaySalesChart.java

```diff
@@ -0,0 +1,138 @@
+            {
+                data.setValueTextSize(7f);
+                data.setValueTextColor(Color.BLACK);
+            }
+            setData(data);
+        }
+
+        SpannableString spannableString = generateCenterText(todayActualSales, todayTargetSales,salesPerformancePercentage, salesGrowthPercentage);
+        setCenterText(spannableString);
+
+        animateY(1400, Easing.EaseInOutQuad);
+        invalidate();
+    }
+
+    // Method to generate customizable center text
+    private SpannableString generateCenterText(float todayActualSales, float todayTargetSales, float overallSalesPercentage, float salesGrowthPercentage) {
+
+        @SuppressLint("DefaultLocale") String centerText = String.format(
+            "Summary\n"
+                +"%.0f%% to Goal\n"
+                +"Actual Sales: %s\n"
+                +"Target Sales: %s\n"
+                +"+%.2f%% vs Yesterday",
+            overallSalesPercentage,
+            StringUtils.formatPesoPrefix(todayActualSales),
+            StringUtils.formatPesoPrefix(todayTargetSales),
+            salesGrowthPercentage
+        );
+
+        SpannableString spannableString = new SpannableString(centerText);
+        {// Change the color of "Today Sales" and "10%" to green
+            int blueColor = Color.parseColor("#0000FF");
+            int greenColor = Color.parseColor("#40C94F");
+            int crimsonColor = Color.parseColor("#DC143C");
+
+            spannableString.setSpan(new ForegroundColorSpan(blueColor), 0, 7, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+            spannableString.setSpan(new StyleSpan(Typeface.BOLD), 0, 7, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+        }
+
+        return spannableString;
+    }
+
+    // Method to apply SpannableString for text styling
+    private SpannableString getSpannableString(String text) {
+        SpannableString spannableString = new SpannableString(text);
+
+        // Change the color of "Today Sales" and "10%" to green
+        int blueColor = Color.parseColor("#0000FF");
+        int greenColor = Color.parseColor("#40C94F");
+        int crimsonColor = Color.parseColor("#DC143C");
+
+
+        int startIndex = text.indexOf("Today Sales");
+        int endIndex = startIndex + 11; // Length of "Today Sales"
+        spannableString.setSpan(new ForegroundColorSpan(blueColor), startIndex, endIndex, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+        spannableString.setSpan(new StyleSpan(Typeface.BOLD), startIndex, endIndex, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+
+        startIndex = text.indexOf("10%");
+        endIndex = startIndex + 3; // Length of "10%"
+        spannableString.setSpan(new ForegroundColorSpan(greenColor), startIndex, endIndex, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+
+        startIndex = text.indexOf("64%");
+        endIndex = startIndex + 3; // Length of "64%"
+        spannableString.setSpan(new ForegroundColorSpan(crimsonColor), startIndex, endIndex, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Comment:
use this method for reference of finishing the previous comment I made.
After finishing the implementation, please delete this function since it is unused